ember-cli-build.js

```diff
@@ -12,6 +12,8 @@ module.exports = function (defaults) {
     },
   });
 
+  app.import('vendor/jsdWidgetScript.js');
```

Comment:
I doubt this is required if you're sourcing the script directly from atlassian.com in index.html. Either include the js as packaged here in index.html or drop this script.

Reply:
What do you mean exactly @nvdk ?

Reply:
Oh! I must've forgotten to rebuild when I made changes to the build script because I thought it stopped working when I removed either one.